src/pages/SubmissionConfig.js

```diff
@@ -47,7 +49,7 @@ export const SubmissionConfig = ({ methodId }) => {
       const searchResult = await Ajax(signal).Wds.search.post(recordType)
       setRecords(searchResult.records)
     } catch (error) {
-      notify('error', 'Error loading WDS records', { detail: await (error instanceof Response ? error.text() : error) })
+      setNoRecordData('No existing record')
```

Comment:
- I think it's the record _type_ that's not found, but I don't think users think in terms of "record type" so maybe we can just call it a data table?
- To give users a clue what to do next, can we include the table we were trying to look for here?
```suggestion
      setNoRecordData(`Data Table not found: ${recordType}`)
```